trio/testing/_check_streams.py

```diff
@@ -67,18 +79,18 @@ async def check_one_way_stream(stream_maker, clogged_stream_maker):
         assert isinstance(s, SendStream)
         assert isinstance(r, ReceiveStream)
 
-        async def do_send_all(data):
-            with assert_checkpoints():
-                assert await s.send_all(data) is None
+        async def do_send_all(data: bytes) -> None:
```

Comment:
If `s` is a Stream object here, it actually accepts `bytes | bytearray | memoryview` apparently.

Reply:
It might, but it doesn't really matter - these are internal nested functions, they're only being called with `bytes` objects.